src/modules/roc_fec/reader.cpp

```diff
@@ -95,17 +100,20 @@ packet::PacketPtr Reader::read_() {
                 source_ = pp->rtp()->source;
                 has_source_ = true;
             }
+            cur_block_n_ = pp->fec()->source_block_number;
             cur_block_sn_ = pp->rtp()->seqnum;
-            skip_repair_packets_();
+            drop_outdated_repair_packets_();
         }
 
         if (!pp || pp->fec()->repair_symbol_id > 0) {
             return source_queue_.read();
         }
 
         roc_log(LogDebug, "fec reader: got first packet in a block, start decoding:"
-                          " n_packets_before=%u blk_sn=%lu",
-                n_packets_, (unsigned long)cur_block_sn_);
+                          " n_packets_before=%u block_n=%lu sn=%lu",
```

Comment:
I suggest to use "sbn" abbreviation when we refer to source block number in logs.

Reply:
Done